gitmanager.py

```diff
@@ -260,7 +260,7 @@ def add_to_blacklist(cls, blacklist='', item_to_blacklist='', username='', chat_
         if op == 'blacklist':
             return (True, "Blacklisted `{0}`".format(item))
         elif op == 'watch':
-            return (True, "Added `{0}` to watchlist".format(item))
+            return (True, "`{0}` will be added to the watchlist once the ongoing CI testing is done".format(item))
```

Comment:
This shouldn't change. The code in gitmanager.py only knows that the item was, in fact, added to the watchlist. It has no information about if the rest of the system thinks that it's acceptable to reload the blacklists and/or reboot.

Reply:
Done, that was what I originally suggested, I've reverted that [in this commit](https://github.com/CoconutMacaroon/SmokeDetector/commit/b62860a6a8781f7e74b7f01e6c1d4c879461bb01)